project_wbs/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+Project Work Breakdown Structure
+================================
+
+* The hierarchy of a project is considered the WBS
```

Comment:
This list would be easier to read if split in 2 or 3 main topics:
1. Adding WBS information to Odoo Projects
2. Searching and Browsing WBS